website/pages/docs/k8s/upgrade/index.mdx

```diff
@@ -7,63 +7,303 @@ description: Upgrade Consul on Kubernetes
 
 # Upgrade Consul on Kubernetes
 
-To upgrade Consul on Kubernetes, we follow the same pattern as
-[generally upgrading Consul](/docs/upgrading), except we can use
-the Helm chart to step through a rolling deploy. It is important to understand
-how to [generally upgrade Consul](/docs/upgrading) before reading this
-section.
+## Upgrade Types
 
-Upgrading Consul on Kubernetes will follow the same pattern: each server
-will be updated one-by-one. After that is successful, the clients will
-be updated in batches.
+Consul on Kubernetes will need to be upgraded/updated if your Helm configuration
+is changed, if a new Helm chart is released or if you wish to upgrade your Consul version.
+
+### Helm Configuration Changes
+
+If you make a change to your Helm values file, you will need to run `helm upgrade`
+for those changes to take effect.
+
+For example, if you've installed Consul with
+
+```yaml
+global:
+  name: consul
+connectInject:
+  enabled: false
+```
+
+And you wish to set `connectInject.enabled` to `true`:
+
+```diff
+global:
+  name: consul
+connectInject:
+-  enabled: false
++  enabled: true
+```
+
+Follow the following steps.
+
+1. Determine your current installed chart version
+
+```bash
+helm list -f consul
+NAME  	NAMESPACE	REVISION	UPDATED        	STATUS  	CHART        	APP VERSION
+consul	default  	2       	2020-09-30 ...	deployed	consul-0.24.0	1.8.2
+```
+
+In this example, I'm running version `0.24.0` (from `consul-0.24.0`).
+
+1. Run `helm upgrade consul hashicorp/consul --version 0.24.0 -f /path/to/my/values.yaml`.
```

Comment:
```suggestion
1. Perform a `helm upgrade`:

```bash
helm upgrade consul hashicorp/consul --version 0.24.0 -f /path/to/your/values.yaml
```

```

Reply:
You'll want to double check this one in a local build. Sometimes adding code fences via suggestions, which themselves use fences, confuses the parser.